components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -179,25 +182,34 @@ export default class SubMenuItem extends React.PureComponent<Props, State> {
         if (!isMobile) {
             subMenuContent = (
                 <ul
-                    className={'a11y__popup Menu dropdown-menu SubMenu'}
+                    className={classNames(['a11y__popup Menu dropdown-menu SubMenu', {categorySortingMenu}])}
                     style={subMenuStyle}
                 >
                     {hasSubmenu ? subMenu!.map((s) => {
+                        const hasDivider = s.id === 'SidebarChannelMenu-moveToDivider';
```

Comment:
For this, maybe we could export a constant that causes a divider to be rendered instead of the normal `SubMenuItem`?

Reply:
It actually is more work than one would think because of the submenu and the types it takes, I will need to add more HTML etc. I say we keep it in mind for the new Menu.